packages/imperative/__tests__/__integration__/imperative/__tests__/__integration__/cli/config/secure/__scripts__/secure_prompt.sh

```diff
@@ -4,5 +4,24 @@ FORCE_COLOR=0
 
 mkdir -p test
 cd test
-echo "anotherFakeValue" | imperative-test-cli config secure $1
+values=${IMPERATIVE_TEST_CLI_SECURE_VALUES:-"anotherFakeValue,undefined"}
+command="imperative-test-cli config secure $1"
+
+cat > node_script.js <<EOF
+const cp = require('child_process');
+const command = '$command'.trim().split(' ');
+const child = cp.spawn(command.shift(), command, { stdio: "pipe" });
+const values = "$values".split(',');
+child.stdout.on('data', (data) => {
+  console.log(data.toString());
+  if (data.toString().includes("Press ENTER to skip:") || data.toString().includes("(will be hidden):")) {
+    child.stdin.write(values.shift() + '\n');
+    if (values.length === 0) {
+      child.stdin.end();
+    }
+  }
+});
```

Comment:
I see the same logic is in a couple places and it seems quite useful for evaluating multiple prompts in integration tests. What are your thoughts on de-duping and generalizing this? I was thinking it could be beneficial in its own file as a shell function with a few params (command, values) and the file could be `source`d in the test scripts 😋